widgets/Salesforce-batchsink.json

```diff
@@ -193,6 +193,22 @@
             ],
             "default": "Skip on error"
           }
+        },
+        {
+          "widget-type": "toggle",
+          "label": "Datatype Validation",
+          "name": "datatypeValidation",
+          "widget-attributes": {
+            "on": {
+              "value": "true",
+              "label": "YES"
+            },
+            "off": {
+              "value": "false",
+              "label": "NO"
+            },
+            "default": "false"
```

Comment:
Should we have it "true" here to ensure any new pipeline has it enabled?

Reply:
Proper way would be to make it true by default, but it will impact the existing customer if they upgrade their pipelines.
Same was also mentioned by Pandiyan in the bug to keep it false for backward compatibility.

Reply:
+1 please test this case manually as well. 
1. Pipeline upgrade: salesforce sink v1.4.7 -> v1.4.8 - data type validation should be disabled
2. Create a new pipeline with salesforce sink v1.4.8 with default settings for data type validation => data type validation should be enabled

Reply:
This test case is tested.